index.html

```diff
@@ -429,48 +429,51 @@ <h2 class="faqs-heading">FAQS</h2>
     </main>
     <footer class="page-footer">
         <section class="join-our-community">

```

Comment:
On the homepage, `.join-our-community` has different styles from other pages and shouldn't have a gradient.